school_evaluation/models/school_evaluation.py

```diff
@@ -35,9 +21,47 @@ def _compute_total_points(self):
                 rec.total = sum(line.point_id.rating for line in rec.eval_line
                                 if line.point_id.rating)
 
+    student_id = fields.Many2one('student.student', 'Student Name',
+                                 help="Select Student")
+    teacher_id = fields.Many2one('school.teacher', "Teacher",
+                                 help='Select teacher')
+    type = fields.Selection([('student', 'Student'),
+                             ('faculty', 'Faculty')],
+                            'User Type', required=True,
+                            help="Type of evaluation")
+    date = fields.Date('Evaluation Date', required=True,
+                       help="Evaluation Date",
+                       default=fields.Date.context_today)
+    eval_line = fields.One2many('school.evaluation.line', 'eval_id',
```

Comment:
Field name should be eval_line_ids

Reply:
Yeas ma'am as per standard naming it should be , but if it would be changed then it can cause the issue as that field might used in the other modules and remote methods. Should it be updated?

Reply:
It should update everywhere, @AnuKPatel @JayVora-SerpentCS what's your opinion in this?